operators/index.js

```diff
@@ -30,6 +30,18 @@ function type(value) {
   })
 }
 
+// We don't need the author "prefix" to be an actual prefix, it can just be any
+// predefined positions in the "information" part of the author ID.
+//
+// WARNING: when updating this, be extra careful that the resulting number isn't
+// larger than the smallest ID's length. E.g. classic feed IDs are 53 characters
+// long, and the base64 part ends at character 44, so AUTHOR_PREFIX_OFFSET must
+// be smaller than 44.
```

Comment:
This is pretty clever. Nice one :)

Reply:
if I were to nitpick (seeing as I can't find anything else to comment on ;-)), maybe mention here that it must be smaller than 44 - prefix bits in bytes, so by default less than 40.